lib/collection/src/operations/conversions.rs

```diff
@@ -1613,20 +1613,16 @@ impl TryFrom<api::grpc::qdrant::CreateShardKey> for CreateShardingKey {
                 .map(NonZeroU32::try_from)
                 .transpose()
                 .map_err(|err| {
-                    Status::invalid_argument(format!("Replication factor cannot be zero: {err}"))
+                    Status::invalid_argument(format!("Shard number factor cannot be zero: {err}"))
```

Comment:
Just "shard number"?

```suggestion
                    Status::invalid_argument(format!("Shard number cannot be zero: {err}"))
```

Reply:
Thanks